ironfish/src/blockchain/blockchain.ts

```diff
@@ -406,6 +406,8 @@ export class Blockchain {
 
         this.resolveOrphans(block)
 
+        this.metrics.chainDatabaseSize.value = await this.db.size()
```

Comment:
I am logging the chain db size when adding a new block. I am not sure if this will affect the chain performance. Let me know if there is better place to do it.

Reply:
What about disconnecting a block?

Reply:
Added for removeBlock()